static/src/stylesheets/module/_adslot.scss

```diff
@@ -414,6 +414,11 @@
     padding: 0;
     margin: 0;
 
+    &.ad-slot--carrot {
```

Comment:
Does this need to be nested? Could we move it to after `ad-slot--fluid`? That way we keep the scss flatter, and can remove the ampersand:

```
    .... 
} // end of ad-slot--fluid

.ad-slot--carrot {
    min-height: 0;
    margin-bottom: 16px;
}

.ad-slot--fabric-v1 {
    min-height: 250px;
}
```